storm-client/src/jvm/org/apache/storm/messaging/netty/BackPressureStatus.java

```diff
@@ -28,19 +28,21 @@
 import java.util.concurrent.atomic.AtomicLong;
 
 // Instances of this type are sent from NettyWorker to upstream WorkerTransfer to indicate BackPressure situation
-public class BackPressureStatus implements java.io.Serializable {
-    static final long serialVersionUID = 1L;
+public class BackPressureStatus {
     public static final short IDENTIFIER = (short)-600;
     private static final int SIZE_OF_ID = 2; // size if IDENTIFIER
     private static final int SIZE_OF_INT = 4;
 
     private static AtomicLong bpCount = new AtomicLong(0);
 
-    public final String workerId;
+    public String workerId;
     public final long id;                       // monotonically increasing id
```

Comment:
Just wondering: the characteristic of `monotonically increasing` guarantee `id` to be unique in a worker, but not among workers, and also reset to 0 after worker crash and restart. Does it hurt the backpressure logic at any chance?